package.json

```diff
@@ -7,10 +7,19 @@
     "dev": "./node_modules/.bin/rollup -c -w",
     "build": "npm run prettier:fix && ./node_modules/.bin/rollup -c",
     "test": "jest",
-    "prettier:fix": "./node_modules/.bin/prettier --config ./.prettierrc.js --write 'src/**/*.{ts,tsx}' 'stories/**/*.{ts,tsx}'",
+    "test:watch": "npm t -- --watch",
     "storybook": "./node_modules/.bin/start-storybook -p 6006",
     "storybook:build": "./node_modules/.bin/build-storybook",
-    "type-check": "echo \"type-checking...\" && tsc --noEmit"
+    "type-check": "echo \"type-checking...\" && tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "eslint": "./node_modules/.bin/eslint --ext .jsx,.js,.tsx,.ts src/",
+    "eslint:fix": "./node_modules/.bin/eslint --fix --ext .jsx,.js,.tsx,.ts src/",
+    "stylelint": "./node_modules/.bin/stylelint \"src/**/*.{css,scss}\"",
+    "stylelint:fix": "npm run stylelint -- --fix",
+    "prettify": "./node_modules/.bin/prettier --config ./.prettierrc.js --write \"src/**/*.{ts,tsx}\" \"stories/**/*.{ts,tsx}\"",
+    "lint:fix": "npm run stylelint:fix && npm run eslint:fix",
+    "lint": "npm run stylelint && npm run eslint",
+    "checkPackageUpdates": "npx npm-check-updates"
```

Comment:
```suggestion
    "check-package-updates": "npx npm-check-updates"
```

Reply:
I guess it the same on the [project-starter](https://github.com/Hipo/hipo-frontend-project-starter/blob/master/package.json). I can open a PR for this